tests/testthat/test-object-package.R

```diff
@@ -0,0 +1,34 @@
+context("Object: Package")
+
+test_that("author with one given name ", {
+  expect_equal(author_desc(list(given="Given",
```

Comment:
Can you please use the line breaking/identing rules described at http://style.tidyverse.org/syntax.html#long-lines ?

You can use [styler](https://github.com/krlmlr/styler) to do so automatically.

Reply:
Done